test/Core.Test/Utilities/StaticStoreTests.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Bit.Core.Enums;
+using Bit.Core.Utilities;
+
+namespace Bit.Core.Test.Utilities;
+using Xunit;
+
+public class StaticStoreTests
+{
+    [Fact]
+    public void StaticStore_Initialization_Success()
+    {
+        var plans = StaticStore.Plans;
+        Assert.NotNull(plans);
+        Assert.NotEmpty(plans);
+        Assert.Equal(17, plans.Count());
+    }
+
+    [Fact]
+    public void StaticStore_GetPlanByPlanType_Success()
```

Comment:
Just a nitpick but could be converted to `[Theory]` and then provide the `PlanType` as `InlineData`